src/components/custom-field-input-date/custom-field-input-date.test.jsx

```diff
@@ -45,25 +45,32 @@ describe('src/components/custom-field-input-date/custom-field-input-date', () =>
     it('has a name attribute; returns name prop', () => {
       const ref = createRef();
       render(<CustomFieldInputDate {...requiredProps} ref={ref} />);
-      expect(ref.current.name).toEqual('field-id');
+      expect(ref.current.name).toEqual('field-name');
     });
 
     it('has a value attribute; returns the value in yyyy-mm-dd format', () => {
       const ref = createRef();
-      render(<CustomFieldInputDate {...requiredProps} ref={ref} value={'09/27/2020'} />);
+      render(<CustomFieldInputDate {...requiredProps} ref={ref} value="2020-09-27" />);
       expect(ref.current.value).toEqual('2020-09-27');
     });
```

Comment:
One thing I liked about the older format of the test was that it proved that even if you gave it a date in the american date format, it still returned it in ISO8601.

Is that still the case? If so, can we revert the change? If it's not the case, can we update the test name?

Reply:
yeah, the `value` prop API now is always full-date ISO8601 (i.e. `YYYY-MM-DD`). 

We should always user ISO8601 full-date or date-time for programming bits purposes (or native Date functionality).

Reply:
I think the test name is still consistent. Ideally, I would just go even simpler:

```
it('has a string value')
```

The implementation details are up to the tests themselves.